releng/tools/rmake

```diff
@@ -336,6 +337,17 @@ def tidy(patterns):
         sys.exit(1)
 
 def main():
+    maxThreads = 2 + multiprocessing.cpu_count()
+    for i in range(len(sys.argv)):
+        if sys.argv[i][:2] == "-j":
```

Comment:
skip the first (its the file name) - shouldnt be a problem, but its good practice